brave/brave-core
# Adds detailed view for pending contributions

Resolves https://github.com/brave/brave-browser/issues/3175

UI implementation: https://github.com/brave/brave-ui/pull/360

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- enable rewards
- tip to couple of unverified site
- go to rewards settings page
- click on show all pending contributions in the wallet
- try to delete some of them
- try clicking on remove all and make sure that all are removed

## Reviewer Checklist:
